ChangeLog.md

```diff
@@ -21,14 +21,16 @@ Current Trunk
   you add, and does not need to contain the list of default system imports like
   ``emscripten_sleep``. There is no harm in providing them, though, so this
   is not a breaking change.
-- Enable DWARF support: with building with -g, normal DWARF emitting happens,
-  and when linking with -g we preserve that and update it. This is a change
+- Enable DWARF support: When compiling with `-g`, normal DWARF emitting happens,
+  and when linking with `-g` we preserve that and update it. This is a change
   from before, where we assumed DWARF was unneeded and did not emit it, so this
   can increase the size of debug builds (i.e. builds compiling and/or linking
   with -g). This change is necessary for full debugging support, that is, to
-  be able to build with -g and use a debugger. Before this change only the
-  -gforce_dwarf flag enabled DWARF; that flag is now removed. For more info
-  and background see #10325.
+  be able to build with `-g` and use a debugger. Before this change only the
+  `-gforce_dwarf` flag enabled DWARF; that flag is now removed. If you want
+  the old behavior, build your object files with `-gline-tables-only` (which
```

Comment:
I would change the parenthetical statement to
"(this will only add line table info, which is just enough for things like source maps and does not include full debug info)"
i.e. I would avoid "full DWARF" because line tables are still DWARF

Reply:
Sounds good, fixed.